packages/UI/src/components/organisms/cards/ShareCard/index.ts

```diff
@@ -0,0 +1,8 @@
+export { default } from './ShareCard';
+
+export interface ShareCardProps {
+  title: 'idea' | 'position' | 'product';
+  platforms: string;
```

Comment:
@alston-hsu
We have specific `platforms` that we support in this regard, let's make this prop more limited by actually defining the strings one by one.

Edit: is this prop even used?

Reply:
Done